### DISCUSSION TITLE
Transfer a domain

### DISCUSSION THREAD

**Discussion by Iznobadd** [vercel/community#5246] [Help] [answered]
### Summary

Hello, I'm reaching out because I currently have a domain name with LWS that I would like to transfer. However, when I request the transfer on Vercel, I'm told this: 'Domain is transferable with a 1 year renewal for $20.' This is despite having just renewed the domain name. Thank you in advance for your response.

### Steps to Reproduce

Transfer a domain

**Comment by amyegan** [comment, marked as answer]
Good question @Iznobadd. This is an industry-standard practice when transferring domains. When a domain is transferred, you pay the renewal price early so an additional year is added to the expiration date of the domain. It's done this way to ensure that registrars get paid while also making sure customers don't pay anything extra for domain transfers.

**Reply by amyegan** [reply]
You can also use a custom domain with your Vercel project without transferring it. Just update the DNS records or nameservers as described here: https://vercel.com/docs/projects/domains/add-a-domain

**Comment by melotek** [comment]
Ok you trying to tell this is standard? I remove all projects your comany pretend to be scam, i am pretty sure you break the law here.

**Reply by amyegan** [reply]
Sorry for the confusion! Please let me clarify. **You do not need to transfer a domain to use it with a Vercel project.** It can be [configured using DNS settings](https://vercel.com/docs/projects/domains/add-a-domain#configure-the-domain) with a different registrar.

The only time you would pay for a custom domain on your project is if you purchase a new one or transfer one in for Vercel to manage. It is normal to **extend the domain expiration date by one year** when transferring in a domain from another registrar. The associated renewal fee is charged at the time of transfer. This is not a scam or an additional fee.

Most domain registrars follow the same procedure.

https://www.pcmag.com/how-to/how-to-transfer-a-domain-name#4
<img width="721" alt="you will likely need to renew your domain name with the new domain host" src="https://github.com/user-attachments/assets/c3ff4802-22ed-42f9-98ba-3482bc39b27b">


https://www.techradar.com/how-to/how-to-transfer-a-domain-name
<img width="617" alt="Most providers will require you to pay for a year of service" src="https://github.com/user-attachments/assets/4006feb5-0fa3-4c36-aa69-6171b6af7558">
